pcsx2/gui/Debugger/CtrlRegisterList.cpp

```diff
@@ -20,505 +20,436 @@
 #include "DebugEvents.h"
 #include "AppConfig.h"
 
-BEGIN_EVENT_TABLE(CtrlRegisterList, wxWindow)
-	EVT_PAINT(CtrlRegisterList::paintEvent)
-	EVT_LEFT_DOWN(CtrlRegisterList::mouseEvent)
-	EVT_RIGHT_DOWN(CtrlRegisterList::mouseEvent)
-	EVT_RIGHT_UP(CtrlRegisterList::mouseEvent)
-	EVT_MOTION(CtrlRegisterList::mouseEvent)
-	EVT_KEY_DOWN(CtrlRegisterList::keydownEvent)
-END_EVENT_TABLE()
+
 
 enum DisassemblyMenuIdentifiers
 {
-	ID_REGISTERLIST_DISPLAY32 = 1,
-	ID_REGISTERLIST_DISPLAY64,
-	ID_REGISTERLIST_DISPLAY128,
-	ID_REGISTERLIST_CHANGELOWER,
-	ID_REGISTERLIST_CHANGEUPPER,
-	ID_REGISTERLIST_CHANGEVALUE
+    ID_REGISTERLIST_DISPLAY32 = 1,
+    ID_REGISTERLIST_DISPLAY64,
+    ID_REGISTERLIST_DISPLAY128,
+    ID_REGISTERLIST_CHANGELOWER,
+    ID_REGISTERLIST_CHANGEUPPER,
+    ID_REGISTERLIST_CHANGEVALUE
 };
 
+CtrlRegisterList::CtrlRegisterList(wxWindow* parent, DebugInterface* _cpu) :
+    wxWindow(parent, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxWANTS_CHARS | wxBORDER_NONE),
+    cpu(_cpu),
+    lastPc(0),
+    maxBits(128),
+    lastCycles(0),
+    needsValueUpdating(true),
+    needsSizeUpdating(true)
```

Comment:
Initialisation triggers warnings when using gcc - Swap maxBits and lastCycles lines, swap needsValueUpdating and needsSizeUpdating lines.